homeassistant/components/fritz/switch.py

```diff
@@ -561,11 +561,17 @@ def __init__(self, fritzbox_tools: FritzBoxTools, device: FritzDevice) -> None:
         """Init Fritz profile."""
         super().__init__(fritzbox_tools, device)
         self._attr_is_on: bool = False
+        self._device_name = device.hostname
+
+    @property
+    def name(self) -> str:
+        """Return switch name."""
+        return f"{self._device_name} Internet Access"
```

Comment:
This would override the parent:
https://github.com/home-assistant/core/blob/a3126ae7854d0ca8ec77c20ae734e0b76b924f0c/homeassistant/components/fritz/common.py#L267-L270

Could you remove the parent instead and also add the name property to the device tracker?

Reply:
Small cleanup